recipes/quill/build.sh

```diff
@@ -0,0 +1,8 @@
+cmake ${CMAKE_ARGS} \
+  -DCMAKE_PREFIX_PATH:PATH=${PREFIX} \
+  -DCMAKE_INSTALL_PREFIX:PATH=${PREFIX} \
+  -DCMAKE_INSTALL_LIBDIR=lib \
+  -DCMAKE_BUILD_TYPE=Release \
+  .
+
```

Comment:
Is there a `make check` or `ctest` that we can run here and in the bat as well?

Reply:
I have added it. For mac os there is an error for `MAP_ANONYMOUS` but the correct file `<sys/mman.h>` is already included. Also the repo#s CI passes all tests for macos [here](https://github.com/odygrd/quill/actions/runs/9338908540)

Can you advise on how i can get it resolved so maybe how i can skip running it for macos ?

Reply:
thanks, is there an `ENV` flag that  I can check inside the build.sh that indicates it is running on macos ?

Reply:
You can probably use an if-clause like:

```shell
if [[ "$(uname)" == "Darwin" ]]; then
  do somthing
fi
``

Reply:
thanks you, I have done this, I will open an issue on the feedstock repo and will try to resolve it

Reply:
I've successfully applied a patch to resolve the issue : )